CIP-XXXX/README.md

```diff
@@ -0,0 +1,179 @@
+---
+CIP: ????
```

Comment:
```suggestion
CIP: ?
```

Reply:
(as specified in CIP-0001)

Reply:
Done. FYI, this syntax appears to cause an error message in the rendered version:
![image](https://github.com/cardano-foundation/CIPs/assets/20246757/201d3a7b-1cdf-4529-9496-9a8bf9c75741)

It shows up in other PRs as well: https://github.com/serejke/CIPs/blob/CIP-keccak256/CIP-%3F%3F%3F%3F/README.md